npm_tasks/explore.js

```diff
@@ -0,0 +1,34 @@
+// This is meant to provide a cross-platform way of doing the same thing
+// as "npm explore <name> -- <cmd>" (which doesn't work on Windows; see
+// https://github.com/npm/npm/issues/8932).
+//
+// Note that <cmd> is expected to be something extremely simple like
+// "npm install" or "git pull origin master". It shouldn't contain any
+// quoted arguments or other funky things.
```

Comment:
This is an annoying limitation and we should probably get rid of it if we decide to turn this into an npm module or something. Um, assuming there isn't already an npm module that does the same thing. :eyes: